@xen-orchestra/lite/src/components/infra/InfraHostItem.vue

```diff
@@ -58,6 +62,14 @@ const isCurrentHost = computed(
   () => props.hostOpaqueRef === uiStore.currentHostOpaqueRef
 );
 const [isExpanded, toggle] = useToggle(true);
+
+const getVmCount = computed(() => {
+  const { recordsByHostRef } = useVmCollection();
```

Comment:
Move `const { recordsByHostRef } = useVmCollection()` outside of the computed. otherwise `useVmCollection` will be called every time the computed is triggered.